plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -170,6 +170,77 @@ func (n *NodeLabelPrioritizer) CalculateNodeLabelPriority(pod *api.Pod, machines
 	return result, nil
 }
 
+// This is a reasonable size range of all container images. 90%ile of images on dockerhub drops into this range.
+const minImgSize int64 = 23000000   // min 23MB
+const maxImgSize int64 = 1000000000 // max 1000MB
```

Comment:
Also, can be combined into

const(
        mb uint64 = 1024 * 1024
        minImgSize uint64 = 23 * mb
        maxImgSize uint64 = 1000 * mb
)

)